src/main/java/io/turntabl/dataformatting/TCMS.java

```diff
@@ -0,0 +1,20 @@
+package io.turntabl.dataformatting;
+
+public class TCMS {
+    public static void main(String[] args) {
+       // ClientDetails clientDetails = new ClientDetails("marfo kofi","kkk","sdfghjkl");
+        ;
+        ClientDetails malam = new ClientDetails("alex owusu", "accra-gh", "alex@gmail.com");
+       System.out.println(ClientDetails.printDetails());
+        ClientDetails francis = new ClientDetails("francis owusu", "accra-gh", "alex@gmail.com");
+        System.out.println("=========================== ");
+        System.out.println(AnsiConsole.GREEN +"NAME:    " +  malam.getFullname()+'\n'+"ADDRESS: "+ malam.getAddress());
```

Comment:
Don't be using the worry much on the getters, I have a ClientData which we will use to feed this main.
Good Work Here

Reply:
thanks